packages/slice-machine/server/src/api/state.ts

```diff
@@ -109,7 +115,10 @@ export default async function handler(
     ...frontEnv,
     sliceMachineAPIUrl: baseUrl,
     shortId: prismicData.shortId,
-    prismicAPIUrl: prismicData.base,
+    prismicAPIUrl: preferWroomBase(
```

Comment:
why not using base variable here instead of calling again the function ?

Reply:
I can change that back and it should still work now

Reply:
but who know's where it'll be incorrectly used